.eslintrc.json

```diff
@@ -1,5 +1,5 @@
 {
-  "ignorePatterns": ["**/*.js", "dist"],
+  "ignorePatterns": ["**/*.js", "db/**/*"],
```

Comment:
I think you should leave the dist folder too
If you remove it the `npm run lint` will fail if you have built the app on your computer